pkg/cmd/dev/build.go

```diff
@@ -120,6 +121,20 @@ var allBuildTargets = func() []string {
 }()
 
 func (d *dev) build(cmd *cobra.Command, commandLine []string) error {
+	// tmpDir will contain the build event binary file if produced.
+	tmpDir, err := os.MkdirTemp("", "")
```

Comment:
Nit: Consider not creating this directory at all (and not cleaning it up) if `buildutil.CrdbTestBuild` is set.